clients/include/utility.hpp

```diff
@@ -1627,6 +1627,169 @@ inline void host_gebsr_to_csr(hipsparseDirection_t    direction,
     }
 }
 
+// template <typename T>
+// inline void host_csr_to_gebsr(hipsparseDirection_t    direction,
+//                               int                     M,
+//                               int                     N,
+//                               int                     row_block_dim,
+//                               int                     col_block_dim,
+//                               int&                    nnzb,
+//                               hipsparseIndexBase_t    csr_base,
+//                               const std::vector<int>& csr_row_ptr,
+//                               const std::vector<int>& csr_col_ind,
+//                               const std::vector<T>&   csr_val,
+//                               hipsparseIndexBase_t    bsr_base,
+//                               std::vector<int>&       bsr_row_ptr,
+//                               std::vector<int>&       bsr_col_ind,
+//                               std::vector<T>&         bsr_val)
+// {
+//     int mb = (M + row_block_dim - 1) / row_block_dim;
+//     int nb = (N + col_block_dim - 1) / col_block_dim;
+
```

Comment:
why in comment?

Reply:
There is a bug in gebsr2gebsr that is causing hipsparse to fail on CI but not on my machine. I am going through and trying to find it